special-interest-groups/build-interest-group/README.md

```diff
@@ -0,0 +1,24 @@
+# Build Interest Group (BIG)
+
+## Mission Statement
+The Build Interest Group (BIG) overseas the health of OpenSearch repositories by guiding repository management, aligning project standards, and providing informed recommendations to the TSC.
+
+[Charter](./charter.md)
+
+### Membership
+
+We follow the membership guidelines proposed in [SIGs README](../README.md) to add / remove / amend / retire members for BIG.
+
+| Maintainer         | GitHub ID                                               | Affiliation |
```

Comment:
Nitpick, but I'd suggest using "Member" instead of "Maintainer" here

Reply:
Thanks @andrross updated.